src/i18n/en/templates/records.ts

```diff
@@ -4,6 +4,6 @@ export default {
     propagation: "Why do I get different values on my local system?",
     learnHow: "Learn how to set {record} records with your DNS/registrar.",
     propagationNote: "Google DNS is reporting different values. This potentially means that your values have not propagated fully yet.",
-    srvFormat: `Expecting to see an SRV record here? Make sure you're looking at the right sub-domain.
-    <br/>SRV record names are normally formatted as follows: <code>_&lt;service&gt;._&lt;protocol&gt;.name.</code>`,
+    srvTlsaFormat: `Expecting to see an {record} record here? Make sure you're looking at the right sub-domain.
+    <br/>{record} record names are normally formatted as follows: <code>_&lt;service&gt;._&lt;protocol&gt;.name.</code>`,
```

Comment:
This is incorrect. TLSA records are `_<port>._<protocol>.name.`

Reply:
Yep, good spot. Just pushed a fix for this.